src/store/store.ts

```diff
@@ -0,0 +1,25 @@
+import { configureStore } from '@reduxjs/toolkit';
+import createSagaMiddleware from 'redux-saga';
+
+import { IS_DEV } from '@utils';
+
+import rootReducer from './reducer';
+import rootSaga from './sagas';
+
+const sagaMiddleware = createSagaMiddleware();
+const store = configureStore({
+  reducer: rootReducer,
+  middleware: [sagaMiddleware]
```

Comment:
Consider using the middleware built-in to Redux Toolkit, as it can help prevent some bugs.

```suggestion
  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(sagaMiddleware)
```

Reply:
Done